openmls/src/group/public_group/diff.rs

```diff
@@ -200,13 +201,17 @@ impl<'a> PublicGroupDiff<'a> {
     pub(crate) fn update_group_context(
         &mut self,
         backend: &impl OpenMlsCryptoProvider,
+        extensions: Option<&Extensions>,
```

Comment:
I suggest to move `Extensions` here and let the caller `clone`. See https://rust-lang.github.io/api-guidelines/flexibility.html#c-caller-control